### ISSUE TITLE
is this still being development?

### ISSUE THREAD

**Issue by nttoanbrvt** [vafour/vafpress-framework#112]
Hi all, I'm looking for a flexible wp framework, I found this, it looks awesome but I see a long time no update.

Just a question, is this still being development?

Thank you.

**Comment by padalec** [comment]
Better use Redux. VP was very nice but no more devlop by the authors. Right now I re-code my FW to Redux too.

**Comment by Zackio** [comment]
Yes I would go with redux or optiontree. This is very nice but it's not
under active development

**Comment by imthat** [comment]
Can we make a branch to continue the work maybe? Dont want to see this framework end like this, cause I saw many devlopers still using it.

**Comment by nttoanbrvt** [comment]
Actually I'm using Redux but it does not has free Metaboxes's addon and another features like this, this is awesome framework but unfortunately it's dead.

**Comment by ghost** [comment]
You can use Admin Page Framework. It has metaboxes free. It is lightweight
and well supported.

**Comment by nttoanbrvt** [comment]
@cmwwebfx it looks promise, thank man.

**Comment by imthat** [comment]
I started a theme using this framework, only to find out the half way this has been gone on idle for a long time. I modified so many things during the process, its quite a pain in the ass, I hope to contribute to this, my only concern is Im not doing it the right way, cause im an ametuer in programming.

**Comment by nttoanbrvt** [comment]
Very hard to continue the work except author come back :)

**Comment by imthat** [comment]
Well indeed :)

**Comment by ghost** [comment]
@nttoanbrvt No problem, hope it does what you are looking for. Something great to note is that you can make Admin Page Framework look however you like.

**Comment by Zackio** [comment]
If you notice there are ton of folks, everyone fixing the same bugs and
improving the same things. If anyone was interested we would need a folk
where several committed users have the ability to commit to the repository.
I've said before I'm happy to be involved - I just have always been a lone
developer so not familiar with how to do this.

**Comment by padalec** [comment]
I use VP in my framework (FW) to create meta boxes (MB) and theme options (when I started using customizer in theme options I have only few fields for theme update etc.) for ShortCodes I using Visual Composer. I very like VP but authors stopped develope so I thinking to move to Redux and buy licence for MB.
@cmwwebfx it's looks very interesting but I'm afraid about I will spend a lot of time to integrated with my FW and authors decided to stop developing just like in VP

**Comment by s43** [comment]
The concept of VPF is really great, it's a brilliant idea to bring an active contributing to keep it surviving cause it really worths it, I'd be happy to be involved too

**Comment by padalec** [comment]
What do you guys think to move VPF to level up or build our own FW:
1. update to PHP 5.3  (namespace, anonymous function etc.)
2. Create Post Types (I have this in my FW)
3. Create Taxonomy (I have this in my FW)
4. Integrate with Customizer (I have this in my FW)
5. Documented all methods and classes
6. Integrate with VC (I have this in my FW)

**Comment by Zackio** [comment]
I think it's a shame for us not to work more together on this - we are
repeating work for ourselves. padalec - I've done some of these things too.
I would be very interested in combining efforts as I do intend on
continuing to use VP in future projects

**Comment by padalec** [comment]
@Zackio grate! I just curious about Your work. Hear is mine: https://github.com/padalec/silverwp . Right now I haven't documentation but I do this in near future.

**Comment by Zackio** [comment]
Michał want to contact me via email? zack.ortus@gmail.com

**Comment by padalec** [comment]
sure!

**Comment by padalec** [comment]
email sended! :)

**Comment by Zackio** [comment]
Hi,

So the framework you sent the link for, is that based on VP or is your own?
I'm taken VP and modified it quite heavily - fixed all the bugs I found and
added functionality.

I think it would be great to combine forces with other devs seems a shame
that so many of us are fixing the same bugs and adding the same
functionality. I've used visual composer and have a class that converts the
VP shortcodes to Visual composer so I can use both. I have a demo in the
options panel.

I think there are a few dedicated enough to either create our own framework
or keep developing Vafpress. I must say, I think vafpress is a little over
complicated and not too easy to figure out whats going on. But the
functionality of the options panel and metaboxes are great.

I will send you my framework.

Curious - what do you do, do you do theme for sale or work for clients? I'm
on Themeforest

**Comment by Zackio** [comment]
I though I received the email but I seemed to be relying the the general
thread instead. I dont' seem to have your email. I've had emails disappear
before

On Thu, May 14, 2015 at 11:56 AM, Zackary Allnutt <zackary.allnutt@gmail.com

> wrote:
> 
> Hi,
> 
> So the framework you sent the link for, is that based on VP or is your
> own? I'm taken VP and modified it quite heavily - fixed all the bugs I
> found and added functionality.
> 
> I think it would be great to combine forces with other devs seems a shame
> that so many of us are fixing the same bugs and adding the same
> functionality. I've used visual composer and have a class that converts the
> VP shortcodes to Visual composer so I can use both. I have a demo in the
> options panel.
> 
> I think there are a few dedicated enough to either create our own
> framework or keep developing Vafpress. I must say, I think vafpress is a
> little over complicated and not too easy to figure out whats going on. But
> the functionality of the options panel and metaboxes are great.
> 
> I will send you my framework.
> 
> Curious - what do you do, do you do theme for sale or work for clients?
> I'm on Themeforest
>

**Comment by padalec** [comment]
Hi, 
yes it's based on VP. 
You have absolutly right!

I sell themes on themeforest to:) and create websites for my clients to based on WP and SilverWP.

**Comment by Zackio** [comment]
Can you try sending me an email again? zack.ortus@gmail.com

**Comment by padalec** [comment]
Yes It's can be you don't have this email because we mailed on different email. Right now I sended to you my work address. Sended once agian

**Comment by Zackio** [comment]
Got your email padalec and replied. It was in spam...

**Comment by wpsoul** [comment]
Hi, guys, I also want to be a part of new VF, because I want to continue use it. Even now, there is no such easy and well documented frameworks like VF. I am not a a good php developer, but I am good frontend developer. Also, I can be a sponsor for creating some modules for framework

**Comment by ayublin** [comment]
Hey Everyone,

In regards of the Vafpress team, I would like to apologize for the inactivity and the lack of communication in those several months. There was some big event in our personal life, and in the end, we decided to disband the team, each of us pursue our own goal now and no one continue to maintain this framework anymore. I should have communicate this way way earlier. Please accept my apologies.

I am grateful to see the interests from all of you to continue on developing Vafpress, let me know if there is anything I can do to help the movement going, but I am afraid I don't have the time to maintain the framework again, also I am not currently in WP development as before, so I am kinda rotten, but I wish I can be of any help in some way. Let me know. Thanks.

**Comment by imthat** [comment]
Glad to see you back bro, now I'm beginning to see some hopes for this project, it's time to joint forces together to revitalize the VF. I'm sure there are a lot of developers thinking the same way. 

You are a valuable part of the community, we would love to see you back. "rotten" is not a good word...what happens in the past rots in the past, every new moment is fresh...

**Comment by ghost** [comment]
Hi guys, great to see you popped in for an explanation of what happened. And yes it is important on what happens in the here and now, not what happened before.

Team, am more interested to see what will happen in future as many are now moving to customizer since the big changes in WordPress requirement that everything should be in customizer now.

**Comment by Zackio** [comment]
Ayub - is it possible for some committed members to take over the project -
we have folks but everyone goes to the offical one. It would be great if we
could commit some of our bug fixes

**Comment by Zackio** [comment]
Also, thanks @ayublin for coming back to explain and also thanks for providing the project in the first place, it was very well done and many of us have made great use of it. Wish you well in what you do now.
